NimiqClientProjects.sln

```diff
@@ -0,0 +1,29 @@
+

```

Comment:
neccessary file?

Reply:
Yes, it's the Visual Studio solution file containing all the projects necessary to run the tests, run the showcase example, and pack the .Net Standard library (aka Nuget package).